apps/server/src/modules/lesson/service/lesson.service.ts

```diff
@@ -24,6 +24,12 @@ export class LessonService {
 		return this.lessonRepo.findAllByCourseIds(courseIds);
 	}
 
+	async findUserDataFromLessons(userId: EntityId): Promise<LessonEntity[]> {
```

Comment:
Should it not named findLessonByUserId? Or similar to other services getLessonByUser() ...i am not really happy to pass the userId from UC to service level. Sounds like no authorization is done before.

Similar point for  finUserDataFromCourseGroup(userId: EntityId)

For this method the public key word is missed.

Reply:
method names changed

I stick to the convention used in the file, additionally if we do not enter the public key it is the default